styleguide/source/_patterns/01-molecules/06-components/11-hub-card/_hub-card.scss

```diff
@@ -42,3 +42,31 @@
   margin: 0 24px;
   order: 2;
 }
+
+.hub-card-portrait {
+  min-height: 300px;
```

Comment:
Ugh, yea... I see what you're saying... The goal was to try to preserve the "portrait-ness" of the card even if there's not much around it, but once the rows were there I see how it'd be hard for it to be shorter